x-pack/plugins/security_solution/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -185,12 +185,12 @@ export const signalRulesAlertType = ({
             );
           }
 
-          const scopedClusterClient = services.getLegacyScopedClusterClient(ml.mlClient);
           // Using fake KibanaRequest as it is needed to satisfy the ML Services API, but can be empty as it is
           // currently unused by the jobsSummary function.
-          const summaryJobs = await (
-            await ml.jobServiceProvider(scopedClusterClient, ({} as unknown) as KibanaRequest)
-          ).jobsSummary([machineLearningJobId]);
+          const fakeRequest = {} as KibanaRequest;
+          const summaryJobs = await ml
```

Comment:
This looks to break ML Rules: when an ML Rule executes, `summaryJobs` is now `undefined`, while on master it is `MlSummaryJob[]`.

I apologize for not having tests that exercise this code: we're in the process of moving to the FTR where we can leverage ML setup helpers, but until then we do not have jobs config/anomalies data to test against.

Reply:
fixed in https://github.com/elastic/kibana/pull/74965/commits/18ccad3df359c6b23f4e69a6103a9bc64eeb133c